data/blog/2024/2024-08-05-using-laminas-continuous-integration.md

```diff
@@ -0,0 +1,129 @@
+---
+id: 2024-08-05-using-laminas-continuous-integration.md
+author: julian
+title: 'Using Laminas Continuous Integration'
+draft: false
+public: true
+created: '2024-08-05T11:00:00-01:00'
+updated: '2024-08-05T11:00:00-01:00'
+tags:
+  - continuous integration
+---
+
+**Continuous Integration (CI)** involves the frequent **merging of code updates** from developers into a shared
+codebase.
+This can be done as often as those updates are ready to be reviewed. Once the CI process is finalized,
+**Continuous Delivery** automatically **deploys the code** to the production environment.
+This flow streamlines the development process for faster and more reliable releases.
+
+<!--- EXTENDED -->
+
+If you only have a handful of codebases, you may be able to handle their maintenance manually, but the automated
+solution is just around the corner. Take the [Laminas Project](https://github.com/laminas), for example.
+
+It has around 200 repositories between the main project and Mezzio. Maintaining all of that code, from bug fixes to
+patches, from improvements to releases is a titanic task. Laminas CI is designed to help maintainers and developers
+review, merge and release updates rapidly and with fewer headaches. Wouldn't you rather sleep soundly at night knowing
+that your code has been checked by a tireless, objective reviewer? Laminas CD will be the topic of a future article, so
+let's focus on Laminas CI.
+
+### Can I use Laminas CI in my own projects?
+
+Of course! Any project or package can benefit from using CI, especially Laminas CI which is designed to be universal
+and easy to install.
+You can grab it from [**Github Marketplace**](https://github.com/marketplace/actions/laminas-continuous-integration).
+
+We will cover the implementation in the next chapter. **DotKernel API** is one of the many organizations that use
```

Comment:
> …is one of the many organizations…

Can we name any others here? A popular name would be helpful.

Reply:
`php/pie` is one example: https://github.com/php/pie/blob/main/.github/workflows/continuous-integration.yml